src/Chapter3.hs

```diff
@@ -907,9 +1005,24 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+newtype Gold = MkGold Int
+
 class Append a where
     append :: a -> a -> a
 
+instance Append Gold where
+    append :: Gold -> Gold -> Gold
+    append (MkGold g1) (MkGold g2) = MkGold (g1 + g2)
+
+instance Append [a] where
+    append :: [a] -> [a] -> [a]
+    append l1 l2 = l1 ++ l2
```

Comment:
Yeap, that is right!
You can eta-reduce this function. As you can see, `append` and `(++)` are actually the same functions, as they return the same output on any inputs 🙂